### PR TITLE
feat(islamic-website): add ar localization

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: apps/islamic-website/components/header/header.tsx [hunk 1]
```diff
@@ -28,29 +29,43 @@ import {
...
   StarIcon,
   QuestionMarkIcon,
+  CheckMarkIcon,
 } from '@haqq/islamic-website-ui-kit';
 import { BurgerMenu } from '../burger-menu/burger-menu';
 import { useMediaQuery } from 'react-responsive';
+import { useLocale, useTranslations } from 'next-intl';
+import { usePathname } from 'next-intl/client';
+
+type localeType = 'en' | 'ar' | 'id';
```
Comment (olegshilov) [inline apps/islamic-website/components/header/header.tsx]:
```suggestion
type LocaleType = keyof typeof localeDisplayNames
```

File: apps/islamic-website/components/header/header.tsx [hunk 2]
```diff
@@ -248,37 +295,46 @@ export function DesktopHeader() {
...
+                locale={locale === 'ar' ? 'ar' : 'en'}
+              >
+                {t('single-links.shariah')}
+              </DesktopHeaderLink>
+              <HeaderDropdown title={t('dropdown-links.about.about')}>
                 <div className="flex gap-x-[18px]">
                   <div className="flex flex-col">
                     <DropdownLink
-                      title="Mission & Vision"
+                      locale={locale === 'ar' ? 'ar' : 'en'}
```
Comment (olegshilov) [inline apps/islamic-website/components/header/header.tsx]:
```suggestion
                      locale={locale}
```

File: apps/islamic-website/components/header/header.tsx [hunk 3]
```diff
@@ -375,32 +442,60 @@ export function DesktopHeader() {
...
               </HeaderDropdown>
+
+              <HeaderDropdown
+                title={
+                  locale === 'ar'
+                    ? 'عربي'
+                    : locale === 'id'
+                    ? 'Bahasa Indonesia'
+                    : 'EN'
+                }
```
Comment (olegshilov) [inline apps/islamic-website/components/header/header.tsx]:
```suggestion
                title={localeDisplayNames[locale]}
```

File: apps/islamic-website/components/layout/layout.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  messages: Record<string, string>;
+  isMobileUserAgent: boolean;
+}>) {
+  const shortLocale = locale ? locale : 'en';
+
+  useEffect(() => {
+    const dir = shortLocale == 'ar' ? 'rtl' : 'ltr';
+
+    document?.querySelector('html')?.setAttribute('dir', dir);
+  }, [shortLocale]);
```
Comment (olegshilov) [inline apps/islamic-website/components/layout/layout.tsx]: why? just set `dir` html attribute below

File: apps/islamic-website/components/layout/layout.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  children,
+  locale,
+  messages,
+  isMobileUserAgent,
+}: PropsWithChildren<{
+  locale: string;
+  messages: Record<string, string>;
+  isMobileUserAgent: boolean;
+}>) {
+  const shortLocale = locale ? locale : 'en';
```
Comment (olegshilov) [inline apps/islamic-website/components/layout/layout.tsx]: why check this? locale is always string.

File: apps/islamic-website/constants.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    icon: <TelegramIcon />,
+    url: 'https://t.me/islamiccoin_int',
+    title: 'Telegram',
+  },
+  {
+    icon: <TwitterIcon />,
+    url: 'https://twitter.com/Islamic_coin',
+    title: 'Twitter',
+  },
+];
```
Comment (olegshilov) [inline apps/islamic-website/constants.tsx]: remove from here and rename back to `constants.ts`

File: libs/islamic-website/community-hub-page/src/assets/images/bg-image-ltr.svg
Comment (olegshilov) [inline libs/islamic-website/community-hub-page/src/assets/images/bg-image-ltr.svg:1]: remove this. you can rotate image with css. just use `scale-x-[-1]` class

File: libs/islamic-website/ui-kit/src/lib/button/button.tsx [hunk 7]
```diff
@@ -26,7 +26,7 @@ export function Button({
   const classNames = clsx(
     'px-[32px] py-[12px] rounded-[8px] h-[48px]',
     'flex justify-center items-center gap-x-[8px]',
-    'text-center uppercase text-[16px] leading-[1.5em] font-[400] font-mono',
+    'text-center uppercase text-[16px] leading-[1.5em] font-[400] font-mono rtl:font-handjet ',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/button/button.tsx]:
```suggestion
    'text-center uppercase text-[16px] leading-[1.5em] font-[400] font-mono rtl:font-handjet',
```

File: libs/islamic-website/ui-kit/src/lib/download-pdf-button/download-pdf-button.tsx [hunk 8]
```diff
@@ -37,7 +37,7 @@ export function DownloadPDFButton({
           </svg>
 
           <div className="flex flex-col text-[12px] font-[400] leading-[1.5em]">
-            <span className="font-mono uppercase">
+            <span className="rtl:font-handjet font-mono  uppercase">
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/download-pdf-button/download-pdf-button.tsx]:
```suggestion
            <span className="rtl:font-handjet font-mono uppercase">
```

File: libs/islamic-website/ui-kit/src/lib/markdown-text/markdown-text.tsx [hunk 9]
```diff
@@ -27,10 +27,11 @@ export function MarkdownText({
         'prose-li:text-[13px] prose-li:leading-[20px] lg:prose-li:text-base',
         'marker:prose-li:font-[600] prose-li:my-0',
         'prose-table:text-[14px] prose-table:leading-[20px] prose-thead:bg-[#2f2f2f]',
-        'prose-th:py-[12px] prose-th:text-left prose-th:font-mono prose-th:uppercase',
-        'first:prose-th:pl-[16px] first:prose-th:pr-0 even:prose-th:px-[24px] last:prose-th:pr-[16px] last:prose-th:pl-0',
-        'first:prose-td:pl-[16px] first:prose-td:pr-0 even:prose-td:px-[24px] last:prose-td:pr-[16px] last:prose-td:pl-0',
+        'prose-th:py-[12px] prose-th:text-left rtl:prose-th:text-right prose-th:font-mono prose-th:uppercase',
+        'first:prose-th:pl-[16px] first:prose-th:pr-0 rtl:first:prose-th:pl-0 rtl:first:prose-th:pr-[16px] even:prose-th:px-[24px] last:prose-th:pr-[16px] last:prose-th:pl-0 rtl:last:prose-th:pl-[16px] rtl:last:prose-th:pr-0',
+        'first:prose-td:pl-[16px] first:prose-td:pr-0 rtl:first:prose-td:pl-0 rtl:first:prose-td:pr-[16px]  even:prose-td:px-[24px] last:prose-td:pr-[16px] last:prose-td:pl-0 rtl:last:prose-td:pl-[16px] rtl:last:prose-td:pr-0',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/markdown-text/markdown-text.tsx]:
```suggestion
        'first:prose-td:pl-[16px] first:prose-td:pr-0 rtl:first:prose-td:pl-0 rtl:first:prose-td:pr-[16px] even:prose-td:px-[24px] last:prose-td:pr-[16px] last:prose-td:pl-0 rtl:last:prose-td:pl-[16px] rtl:last:prose-td:pr-0',
```

File: libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx [hunk 10]
```diff
@@ -59,12 +61,12 @@ export function MemberCard({
                 event.stopPropagation();
               }}
               className={clsx(
-                'text-islamic-primary-green hover:text-islamic-primary-green-hover font-mono uppercase',
+                'text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet font-mono  uppercase',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx]: remove double spaces everywhere

```suggestion
                'text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet font-mono uppercase',
```

File: libs/islamic-website/ui-kit/src/lib/member-modal-card/member-modal-card.tsx [hunk 11]
```diff
@@ -36,7 +36,7 @@ export function MemberModalCard({
         <div className="flex flex-col lg:mt-[24px]">
           <div className="text-[24px] font-[600] leading-[34px]">{title}</div>
           {url && (
-            <div className="text-islamic-primary-green hover:text-islamic-primary-green-hover mt-[8px] flex items-center gap-x-[4px] font-mono uppercase transition-colors duration-300 md:mt-[12px]">
+            <div className="text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet mt-[8px] flex items-center gap-x-[4px] font-mono  uppercase transition-colors duration-300 md:mt-[12px]">
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/member-modal-card/member-modal-card.tsx]:
```suggestion
            <div className="text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet mt-[8px] flex items-center gap-x-[4px] font-mono uppercase transition-colors duration-300 md:mt-[12px]">
```

File: libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx [hunk 12]
```diff
@@ -59,7 +59,7 @@ export function NewsCard({
         <div className="flex flex-col">
           <div
             className={clsx(
-              'mt-[24px] font-mono text-[10px] font-[400] uppercase leading-[16px] text-[#838383] md:text-[11px] md:leading-[18px] lg:text-[12px]',
+              'mt-[24px] font-mono  text-[10px] font-[400] uppercase leading-[16px] text-[#838383] md:text-[11px] md:leading-[18px] lg:text-[12px]',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx]:
```suggestion
              'mt-[24px] font-mono text-[10px] font-[400] uppercase leading-[16px] text-[#838383] md:text-[11px] md:leading-[18px] lg:text-[12px]',
```

File: libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx [hunk 13]
```diff
@@ -85,7 +85,7 @@ export function NewsCard({
         </div>
         <div
           className={clsx(
-            'mt-[16px] font-mono text-[10px] uppercase leading-[16px] text-[#838383] md:text-[11px] md:leading-[18px] lg:text-[12px]',
+            'mt-[16px] font-mono  text-[10px] uppercase leading-[16px] text-[#838383] md:text-[11px] md:leading-[18px] lg:text-[12px]',
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx]: revert

File: apps/islamic-website/app/[locale]/privacy-policy/page.tsx [hunk 14]
```diff
@@ -1,25 +1,26 @@
...
 };
 
-export default async function Page() {
-  const privacyPolicy = await getPrivacyPolicyContent();
+interface PageProps {
+  params: { locale: string };
+}
+
+export default async function Page({ params: locale }: PageProps) {
+  const privacyPolicy = await getPrivacyPolicyContent(locale);
```
Comment (olegshilov) [inline apps/islamic-website/app/[locale]/privacy-policy/page.tsx]: are you sure you get locale correctly?

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/islamic-website/app/[locale]/community-hub/page.tsx [hunk 15]
```diff
@@ -1,6 +1,8 @@
-import { DEPLOY_URL } from '../../constants';
 import type { Metadata } from 'next';
-import { islamicOpenGraphImages } from '../shared-metadata';
+import { DEPLOY_URL } from '../../../constants';
+import { islamicOpenGraphImages } from '../../shared-metadata';
+import { CommunityHubPage } from '@haqq/islamic-website/community-hub-page';
+import { SOCIAL_LINKS } from '../../../components/footer/social-links';
```
Comment (olegshilov) [inline apps/islamic-website/app/[locale]/community-hub/page.tsx]:
```suggestion
import { SOCIAL_LINKS } from '../../../social-links';
```

**Comment by olegshilov** [comment]
@hadzhehsen lets make envglish version first in language tab buttons
image

**Comment by olegshilov** [comment]
@hadzhehsen this sticky menu should have blurred bg

image

**Comment by olegshilov** [comment]
translation for this

image

File: apps/islamic-website/constants.tsx (hunk 6 above)
Comment (hadzhehsen) [reply apps/islamic-website/constants.tsx]: Done

File: apps/islamic-website/components/layout/layout.tsx (hunk 4 above)
Comment (hadzhehsen) [reply apps/islamic-website/components/layout/layout.tsx]: Done

File: libs/islamic-website/community-hub-page/src/assets/images/bg-image-ltr.svg
Comment (hadzhehsen) [reply libs/islamic-website/community-hub-page/src/assets/images/bg-image-ltr.svg:1]: Done

File: libs/islamic-website/ui-kit/src/lib/button/button.tsx (hunk 7 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/button/button.tsx]: Done

File: libs/islamic-website/ui-kit/src/lib/download-pdf-button/download-pdf-button.tsx (hunk 8 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/download-pdf-button/download-pdf-button.tsx]: Done

File: libs/islamic-website/ui-kit/src/lib/markdown-text/markdown-text.tsx (hunk 9 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/markdown-text/markdown-text.tsx]: Done

File: libs/islamic-website/ui-kit/src/lib/member-modal-card/member-modal-card.tsx (hunk 11 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/member-modal-card/member-modal-card.tsx]: Done

File: libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx (hunk 10 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx]: Done

File: libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx (hunk 12 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/src/lib/news-card/news-card.tsx]: Done

File: apps/islamic-website/app/[locale]/privacy-policy/page.tsx (hunk 14 above)
Comment (hadzhehsen) [reply apps/islamic-website/app/[locale]/privacy-policy/page.tsx]: Fixed

File: apps/islamic-website/tailwind.config.js [hunk 16]
```diff
@@ -24,10 +24,26 @@ module.exports = {
       },
       fontFamily: {
         serif: ['var(--font-alexandria)'],
-        mono: ['var(--font-vcr)'],
-        alexandria: ['var(--font-alexandria)'],
```
Comment (olegshilov) [inline apps/islamic-website/tailwind.config.js]: revert

File: libs/islamic-website/shariah-page/src/lib/fatwa-block/fatwa-block.tsx [hunk 17]
```diff
@@ -8,27 +8,30 @@ import fatwaSign3 from '../../assets/images/autographs/autograph-hakim-mohamed.w
...
-  lang: 'ar' | 'en';
+  locale: localeType;
   onClick: () => void;
 }) {
+  const t = useTranslations('shariah-page.fatwa-block.language-buttons');
   return (
     <div
       className={clsx(
-        ' cursor-pointer rounded-[8px] px-[12px] py-[8px] font-mono text-[14px] font-[400] uppercase leading-[20px] text-white transition-colors duration-300',
+        ' rtl:font-handjet font-vcr cursor-pointer rounded-[8px] px-[12px] py-[8px]  text-[14px] font-[400] uppercase leading-[20px] text-white transition-colors duration-300',
```
Comment (olegshilov) [inline libs/islamic-website/shariah-page/src/lib/fatwa-block/fatwa-block.tsx]:
```suggestion
        'rtl:font-handjet font-vcr cursor-pointer rounded-[8px] px-[12px] py-[8px] text-[14px] font-[400] uppercase leading-[20px] text-white transition-colors duration-300',
```

File: libs/islamic-website/ui-kit/.storybook/index.css [hunk 18]
```diff
@@ -33,8 +34,9 @@
   --font-emoji: 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
   --font-family-base: var(--font-apple), var(--font-fallback), var(--font-emoji);
 
-  --font-alexandria: 'Alexandria', var(--font-family-base);
-  --font-mono: 'VCR OSD Mono', var(--font-family-base);
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/.storybook/index.css]: revert

File: libs/islamic-website/ui-kit/tailwind.config.js [hunk 19]
```diff
@@ -21,8 +21,7 @@ module.exports = {
       },
       fontFamily: {
         serif: ['var(--font-alexandria)'],
-        mono: ['var(--font-vcr)'],
-        alexandria: ['var(--font-alexandria)'],
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/tailwind.config.js]: revert

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/islamic-website/tailwind.config.js (hunk 16 above)
Comment (hadzhehsen) [reply apps/islamic-website/tailwind.config.js]: Done

File: libs/islamic-website/shariah-page/src/lib/fatwa-block/fatwa-block.tsx (hunk 17 above)
Comment (hadzhehsen) [reply libs/islamic-website/shariah-page/src/lib/fatwa-block/fatwa-block.tsx]: Done

File: libs/islamic-website/ui-kit/.storybook/index.css (hunk 18 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/.storybook/index.css]: Done

File: libs/islamic-website/ui-kit/tailwind.config.js (hunk 19 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/tailwind.config.js]: Done

File: libs/islamic-website/ui-kit/tailwind.config.js [hunk 20]
```diff
@@ -20,10 +20,9 @@ module.exports = {
         'islamic-bg-black': '#010304',
       },
       fontFamily: {
-        serif: ['var(--font-alexandria)'],
-        mono: ['var(--font-vcr)'],
         alexandria: ['var(--font-alexandria)'],
-        vcr: ['var(--font-vcr)'],
+        handjet: ['var(--font-handjet)'],
+        mono: ['var(--font-vcr)'],
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/tailwind.config.js]:
```suggestion
        vcr: ['var(--font-vcr)'],
```

File: apps/islamic-website/styles/consent-cookie.css [hunk 21]
```diff
@@ -249,7 +249,7 @@
 }
 
 #c-ttl {
-  @apply font-mono uppercase;
+  @apply font-vcr uppercase;
```
Comment (olegshilov) [inline apps/islamic-website/styles/consent-cookie.css]: rtl font

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/islamic-website/styles/consent-cookie.css (hunk 21 above)
Comment (hadzhehsen) [reply apps/islamic-website/styles/consent-cookie.css]: Done

File: libs/islamic-website/ui-kit/tailwind.config.js (hunk 20 above)
Comment (hadzhehsen) [reply libs/islamic-website/ui-kit/tailwind.config.js]: Done

**Review by olegshilov — DISMISSED** [review]

**Review by olegshilov — APPROVED** [review]

**Pull request merged**